dotnet/src/Functions/Functions.UnitTests/OpenApi/RestApiOperationRunnerTests.cs

```diff
@@ -1233,15 +1288,17 @@ public void Dispose()
 
     private sealed class HttpMessageHandlerStub : DelegatingHandler
```

Comment:
Do we really need to have a private Stub ? Can't we use the InternalUtilities for tests and drop this one?

Reply:
Created a seperate task for this https://github.com/microsoft/semantic-kernel/issues/6818